_posts/2020-08-31-release.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 31 August 2020 - FieldTrip version 20200831 has been released
+categories: [news, release]
+tweeted: false
```

Comment:
```suggestion
tweeted: true
tweet: FieldTrip version 20200831 was just released: improvements to NIRS reading, plotting, handling; misc. fixes, and more. Get it http://github.com/fieldtrip/fieldtrip/releases/tag/20200831, detailed log https://github.com/fieldtrip/fieldtrip/compare/20200828...20200831 .
```

This seems like a newsworthy release to me, the others not so much. See my suggested tweet text, which I think might also be good for the website? Possibly in slightly expanded form.

Note by the way that we don't need to explicitly shorten URLs anymore to Twitter, any URL you put in will be automatically shortened and counts as 23 characters. The above suggested tweet text is 187 characters long, taking this into account.

Reply:
I agree that the other there's nothing much worth announcing in the other ones. So maybe the flags can be true , false and NA?

Also if we do switch from automatic tweeting to this more tailored form (something that we should announce!) then what is the need to write the tweet in the script? :) 

Alsoo, do we wanna put on links that direct to Github or to the fieldtrip website e.g. the "download fieldtrip" or the "how do i keep track of the code" tutorial which is more helpful in this case?

tweet: We just released a new FieldTrip version (20200831) with big improvements to NIRS reading, handling and plotting! Get the latest and greatest here: http://www.fieldtriptoolbox.org/download/

news: The fieldtrip team has been working hard to adapt our existing tools for NIRS data! New release (20200831) contains several improvements to NIRS data reading, handling and plotting! Moreover, our plotting functions users the option to plot onto a new figure or not! Check the new release here (http://github.com/fieldtrip/fieldtrip/releases/tag/20200831)

Reply:
> Also if we do switch from automatic tweeting to this more tailored form (something that we should announce!) then what is the need to write the tweet in the script? :)

I think it might still be useful to have a tweet suggested by the person who compiles a noteworthy release, which can then be edited by the tweetmeister (i.e. you) before being sent out. That way, you'll probably know better what are the highlights of any given release than if you have to go by only commit/PR messages alone.

Reply:
Regarding the tweet: why spend 2 times 23 characters to send people to GitHub, if we can also send people to the FieldTrip homepage?

We do not only want to tweet, we want to _communicate_, also to website visitors, and potentially to other current channels (like mailman, facebook) and other future channels (discord, gitter, slack, or whatever will come up in a few years from now).  

I propose a format like  
```
FieldTrip version 20200831 was just released, this includes improvements to processing and plotting NIRS, and more. See http://www.fieldtriptoolbox.org/#31-august-2020
```

And then keep a slightly longer version of the new item on the website that points to the diffs on github and the download page on the website?

Reply:
The (automatic) default tweet could be 
```
FieldTrip version 20200831 was just released XXX. See http://www.fieldtriptoolbox.org/#31-august-2020 
```
where the XXX marks the place to mention a few highlights.

Reply:
I agree about directing people to the fieldtrip website , but then why not direct them to where they would potentially download the new toolbox and learn how to use github to be up to date?

Reply:
I don't know (cannot see) how many people are downloading zip files from github (that is info that MS keeps to itself). I don't know how many people pull using git from github (idem). I do know how many people are downloading from the ftp, that is about 20 per day. Unless I have evidence otherwise, I will continue to work under the assumption that the default (*) for most people is to download from our ftp. 

(*) which is fine with me, and has worked well for 15 years, already prior to github starting as a company, and will work well past MS pulling the plug on github (as google did with their code sharing site) or turning it into something horrible like SourceForge . An yes: please do teach people how to use git and github. I take it that Floris' and Saskia's groups both have a large giuthub user base? I actually don't know where those are (was looking last week with @johalgermissen) but good examples are welcome to complement https://donders-institute.github.io/meg-hackathon/2020-09-03-github-labs/